index.php

```diff
@@ -142,6 +142,12 @@ function get_q_value($mime, $accept)
         error_page('Verification Failed', 'Given Code Was Invalid');
     }
 
+    $response = array('me' => USER_URL);
+
+    if ($scope = (explode(':', $code, 3))[2]) {
```

Comment:
Are those extra brackets on the `explode()` necessary? I don’t see any order-of-execution problems here.